index.html

```diff
@@ -1,11 +1,61 @@
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CALYX</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="style.css">
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer">
+      <div class="navbar bg-primary p-2" style="z-index: 100">
```

Comment:
pls use class for z-index. inline style should be avoided

Reply:
Good to merge.

Reply:
Thanks for pointing it out. Forgot about it